### ISSUE TITLE
Breadcrumbs

### ISSUE THREAD

**Issue by LHinson** [patternfly/patternfly#33]
Introduce breadcrumbs http://www.patternfly.org/pattern-library/navigation/breadcrumbs/#/design

This issue includes: 
- the basic, single line breadcrumb
- breadcrumbs with Title should be addressed. 

Note, the breadcrumb switcher is out of scope for this issue. There is another issue to address that design here: https://github.com/patternfly/patternfly-next/issues/530 (it will be kicked back to interaction design and discovery)

Design Folder: https://drive.google.com/open?id=1sgFcH1cQqiEEitHfpvzAoU44t5AE0LA7 

- [x] Interaction design requirements 
- [x] Visuals 
- [x] First checkpoint complete and approved 
- [ ] CSS 

Contacts: 
- IxD @trahman73 @mcarrano 
- Visual @trahman73 
- CSS @marshmalien

**Comment by cliffpyles** [comment]
PR: https://github.com/patternfly/patternfly-next/pull/37

**Comment by mcarrano** [comment]
Current design can be found here: http://www.patternfly.org/pattern-library/navigation/breadcrumbs/#/design  Note that this design includes something called a "breadcrumb switcher" should consider whether this is a variation of the breadcrumb or a separate component.

**Comment by LHinson** [comment]
@mcarrano I'll file a separate issue for breadcrumb switcher as and enhancement.

**Comment by trahman73** [comment]
[PF4-Breadcrumbs-Spec.pdf](https://github.com/patternfly/patternfly-next/files/2291579/PF4-Breadcrumbs-Spec.pdf)

**Comment by mcarrano** [comment]
Thanks, @trahman73 . @kybaker can you please review?

**Comment by kybaker** [comment]
@trahman73 Made a few comments here. https://redhat.invisionapp.com/share/GYNKIN5SWCU#/314878255_Screen_Shot_2018-08-16_At_2-30-34_PM

**Comment by trahman73** [comment]
Here is an updated spec that incorporates the previous feedback from the group.

[PF4-Breadcrumbs-Spec.pdf](https://github.com/patternfly/patternfly-next/files/2316380/PF4-Breadcrumbs-Spec.pdf)

**Comment by LHinson** [comment]
Nice to see this make it to the final stages! I like the adjustment to dark gray. 

Once this is ready to move onto dev, is @marshmalien interested in picking up the implementation of this?

**Comment by trahman73** [comment]
@LHinson Thanks! I spoke to @marshmalien and we would be interested in picking this up for implementation once the design is approved.

**Comment by kybaker** [comment]
@trahman73 This looks great. My only question is around the spacer below the bar and above the title. I know it reflects the top but after seeing it we may want to reduce that. It is best to judge in the browser though - up to you. I think it is ready to go! Thanks for doing so many revisions.

**Comment by trahman73** [comment]
Thanks @kybaker I agree with your spacing comment so I tried 16px for that line between the breadcrumb and the title. Here is the updated PDF.

[PF4-Breadcrumbs-Spec.pdf](https://github.com/patternfly/patternfly-next/files/2319335/PF4-Breadcrumbs-Spec.pdf)

**Comment by kybaker** [comment]
@trahman73 Looks great! Send it over and I will get it merged in the master Sketch file.

**Comment by LHinson** [comment]
Great, I'm going to go ahead and move this over to @marshmalien to move forward with dev!

**Comment by LHinson** [comment]
Closing this issue since #819 merged.
